projects/driver_controls/src/mechanical_brake_fsm.c

```diff
@@ -0,0 +1,78 @@
+#include "mechanical_brake_fsm.h"
+#include "input_event.h"
+#include "event_arbiter.h"
+#include "log.h"
+
+// Mechanical Brake FSM state definitions
+
+FSM_DECLARE_STATE(state_engaged);
```

Comment:
This comment is just to track the support for mechanical brakes potentially activating regen braking